lib/api/element-commands/waitForElementNotPresent.js

```diff
@@ -36,39 +36,22 @@ class WaitForElementNotPresent extends WaitForElement {
     this.expectedValue = 'not found';
   }
 
-  setupActions() {
-    const validate = (result) => this.isResultSuccess(result) && result.value === null;
-
-    this.executor
-      .queueAction({
-        action: () => this.findElement(),
-        retryOnSuccess: true,
-        retryOnFailure: false,
-        validate,
-        errorHandler: (err) => {
-          if (err.name !== 'TimeoutError') {
-            throw err;
-          }
-
-          if (err.response && err.response.value) {
-            return this.elementFound(err.response);
-          }
-
-          return this.elementNotFound(err.response);
-        }
-      });
-  }
-
   /**
    * Overriding elementFound to fail the test
    *
    * @param response
    * @returns {Promise}
    */
   elementFound(response) {
-    let defaultMsg = 'Timed out while waiting for element <%s> to be removed for %d milliseconds.';
+    const defaultMsg = 'Timed out while waiting for element <%s> to be removed for %d milliseconds.';
 
-    return this.fail(response, 'found', this.expectedValue, defaultMsg);
+    return this.fail({
```

Comment:
We shouldn't change the original response object.

Reply:
`waitForElementPresent` does something similar...
I will change it back.